src/main/java/com/vmtapp/enterprise/service/TicketService.java

```diff
@@ -97,6 +100,7 @@ public Ticket save(Ticket ticket) {
     }
 
     @Override
+    @Cacheable("tickets")
```

Comment:
Since the fetch action may happen multiple times, using a cache can help improve performance. Source:https://martinfowler.com/articles/microservices.html#EvolutionaryDesign